builtin/logical/pki/backend.go

```diff
@@ -0,0 +1,67 @@
+package pki
+
+import (
+	"strings"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+// Factory creates a new backend implementing the logical.Backend interface
+func Factory(map[string]string) (logical.Backend, error) {
+	return Backend(), nil
+}
+
+// Backend returns a new Backend framework struct
+func Backend() *framework.Backend {
+	var b backend
+	b.Backend = &framework.Backend{
+		Help: strings.TrimSpace(backendHelp),
+
+		PathsSpecial: &logical.Paths{
+			Root: []string{
+				"config/*",
+				"revoked/*",
+				"revoke/*",
+				"crl/build",
```

Comment:
Hah, the problem with not yet having unit tests for revocation. I landed on "rotate" so I'll update that here (it's that path in the actual Path struct, just didn't update it here).